src/main/java/org/andersen/homework/model/entity/user/Client.java

```diff
@@ -1,26 +1,27 @@
 package org.andersen.homework.model.entity.user;
 
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.DiscriminatorValue;
+import jakarta.persistence.Entity;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToOne;
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
 import lombok.Setter;
 import lombok.ToString;
 import org.andersen.homework.model.entity.ticket.Ticket;
-import org.andersen.homework.model.enums.UserRole;
 
+@Entity
+@DiscriminatorValue("CLIENT")
 @Getter
 @Setter
+@NoArgsConstructor
 @ToString(callSuper = true)
 @EqualsAndHashCode(callSuper = true)
 public class Client extends User {
 
+  @OneToOne(cascade = CascadeType.REMOVE)
```

Comment:
Why OneToOne relation?

Reply:
One client may have only one ticket, same for ticket. It wasn't specified in the task to make ManyToOne relation so a made it OneToOne. Do i have to change it?

Reply:
No, it's not really matters. Just asking why have you chose it)
In real life seems that each user could have multiple tickets, but if you say that for your project requirements only one - fine for me also :)